content/pages/education/apply.md

```diff
@@ -56,10 +76,9 @@ If you need to make a change (for example, you’re moving to a new school), man
 <div class="usa-alert usa-alert-warning usa-content va-alert" markdown="1">
 	<div class="usa-alert-body">
 
-##### Forms 22-1990E, 22-1990N, 22-5490, and 22-5495
+##### Forms 22-1990N, 22-5490, and 22-5495
 
 You must apply for education benefits using eBenefits if you're:
-- A spouse or dependent
```

Comment:
- A dependent whose sponsor is permanently and totally disabled, making a new application or moving a benefit to a new location
- A dependent whose sponsor is deceased, MIA, or a POW, making a new application or moving a benefit to a new location
- A candidate for the National Call to Service program